include/seqan3/alignment/pairwise/alignment_algorithm.hpp

```diff
@@ -360,7 +443,13 @@ class alignment_algorithm :
             });
 
             if (this->band_touches_last_row())  // TODO [[unlikely]]
-                this->check_score_last_row(get<0>(get<0>(*std::ranges::prev(seqan3::end(col)))), get<3>(cache));
+            {
+                auto [cell, coordinate, trace] = *std::ranges::prev(std::ranges::end(col));
+                std::ignore = trace;
```

Comment:
?

Reply:
to ignore warnings rather do `(void*)trace;`

Reply:
I did some reading and unfortunately it is not intended to be used this way. Also it is really an expressive way to ignore unused variables. So I might as well change it back to ugly.